### PR TITLE
Updates Urist to Byond 513 - 1536

### PR DESCRIPTION
**Pull request by Laniakeaaaa**
- Updates Urist to the most current version of Byond.
- Additionally, for sanity we won't have to downgrade to use the editor now.

This will update Travis to the most recent version as well.

- [x] Fix Arctan reserved word proc

### CODE REVIEW

**Comment by martinlyra** [comment]
I'd like to point out the lastest version right now is `513.1536`

**Comment by Laniakeaaaa** [comment]
I'll reupdate this to the latest version this weekend.

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ jdk: openjdk8
 
 env:
   global:
-    BYOND_MAJOR="512"
-    BYOND_MINOR="1479"
+    BYOND_MAJOR="513"
+    BYOND_MINOR="1534"
```
Comment (martinlyra) [inline .travis.yml:9]: Or have a quick shortcut:
```suggestion
    BYOND_MINOR="1536"
```

File: code/modules/flufftext/TextFilters.dm [hunk 2]
```diff
@@ -135,14 +135,14 @@ proc/RadioChat(mob/living/user, message, distortion_chance = 60, distortion_spee
 					if(english_only)
 						newletter += "*"
 					else
-						newletter = pick("�", "�", "%", "�", "�")
+						newletter = pick("�", "�", "%", "�", "�")
```
Comment (martinlyra) [inline code/modules/flufftext/TextFilters.dm:138]: Encoding b broke
```suggestion
						newletter = pick("ø", "Ð", "%", "æ", "µ")
```

File: code/modules/flufftext/TextFilters.dm [hunk 3]
```diff
@@ -135,14 +135,14 @@ proc/RadioChat(mob/living/user, message, distortion_chance = 60, distortion_spee
...
 				distortion += 0.5 * distortion_speed
 			else if(prob(0.75 * distortion)) // Incomprehensible
 				newletter = pick("<", ">", "!", "$", "%", "^", "&", "*", "~", "#")
 				distortion += 0.75 * distortion_speed
 			else if(prob(0.05 * distortion)) // Total cut out
 				if(!english_only)
-					newletter = "�w��b�%> -BZZT-"
+					newletter = "�w��b�%> -BZZT-"
```
Comment (martinlyra) [inline code/modules/flufftext/TextFilters.dm:145]:
```suggestion
					newletter ="¦w¡¼b»%> -BZZT-"
```

File: code/modules/flufftext/TextFilters.dm [hunk 4]
```diff
@@ -152,22 +152,22 @@ proc/RadioChat(mob/living/user, message, distortion_chance = 60, distortion_spee
 					if("s")
 						newletter = "$"
 					if("e")
-						newletter = "�"
+						newletter = "�"
```
Comment (martinlyra) [inline code/modules/flufftext/TextFilters.dm:155]:
```suggestion
						newletter = "€"
```

File: code/modules/flufftext/TextFilters.dm [hunk 5]
```diff
@@ -152,22 +152,22 @@ proc/RadioChat(mob/living/user, message, distortion_chance = 60, distortion_spee
...
 					if("w")
-						newletter = "�"
+						newletter = "�"
```
Comment (martinlyra) [inline code/modules/flufftext/TextFilters.dm:157]:
```suggestion
						newletter = "ø"
```

File: code/modules/flufftext/TextFilters.dm [hunk 6]
```diff
@@ -152,22 +152,22 @@ proc/RadioChat(mob/living/user, message, distortion_chance = 60, distortion_spee
...
 					if("y")
-						newletter = "�"
+						newletter = "�"
 					if("x")
-						newletter = "�"
+						newletter = "�"
```
Comment (martinlyra) [inline code/modules/flufftext/TextFilters.dm:161]:
```suggestion
						newletter = "æ"
```

File: code/modules/flufftext/TextFilters.dm [hunk 7]
```diff
@@ -152,22 +152,22 @@ proc/RadioChat(mob/living/user, message, distortion_chance = 60, distortion_spee
...
 					if("u")
-						newletter = "�"
+						newletter = "�"
```
Comment (martinlyra) [inline code/modules/flufftext/TextFilters.dm:163]:
```suggestion
						newletter = "µ"
```

File: code/modules/organs/external/_external.dm [hunk 8]
```diff
@@ -1009,7 +1009,7 @@ Note that amputating the affected organ does in fact remove the infection from t
 		W.germ_level = 0
 	return rval
 
-/obj/item/organ/external/proc/clamp()
+/obj/item/organ/external/proc/clampwound()
```
Comment (martinlyra) [inline code/modules/organs/external/_external.dm:1012]: Now, to be pedantic. `clamp_wound` would have been a fancier and better rename.
```suggestion
/obj/item/organ/external/proc/clamp_wound()
```
Don't forget to do this to the three lines in `code/modules/surgery/generic.dm` as well.

**Review by martinlyra — COMMENTED** [review]
Here's my thoughts on what could be changed or improved! Good PR!

**Comment by Laniakeaaaa** [comment]
Thanks, Martin. Relevant changes made, should be good as gold now!

**Review by martinlyra — APPROVED** [review]

**Comment by Glloyd** [comment]
I'll update the server tonight and move it to 513

**Comment by Glloyd** [comment]
Thanks for doing this @Laniakeaaaa

**Pull request merged**